elements/pfe-primary-detail/src/pfe-primary-detail.js

```diff
@@ -300,16 +339,134 @@ class PfePrimaryDetail extends PFElement {
         details: nextDetails
       }
     });
+  } // end _handleHideShow()
 
-    // Set focus to pane
-    const firstFocusableElement = nextDetails.querySelector(
-      'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
-    );
-    if (firstFocusableElement) {
-      firstFocusableElement.focus();
+  /**
+   * A11y features
+   */
+  _isToggle(element) {
+    return element.classList.contains("a11y-toggle");
+  }
+
+  _isPanel(element) {
+    return element.classList.contains("a11y-panel");
+  }
+
+  _allToggles() {
+    return [...this.querySelectorAll(".a11y-toggle")];
```

Comment:
We have an object that has references to the toggles and panels in: `this._slots`
`this._slots.detailsNav` is an array of the elements in order

It's kept up to date in DOM order if anything is up to date.